app/models/comment.rb

```diff
@@ -1,2 +1,10 @@
 class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create_commit :update_comments_counter
+
+  def update_comments_counter
+    post.update(comments_count: post.comments.count)
+  end
```

Comment:
[OPTIONAL]
- [ ] Nice work using the Rails Active callbacks to call the `update_comments_counter` method every time a new comment has been created :clap: However since the `update_comments_counter` method it's only needed in the **Comment** model, it would be nice to make it private to follow ruby best practices.